wandb/data_types.py

```diff
@@ -519,9 +527,10 @@ def seq_to_json(cls, threeD_list, run, key, step):
         jsons = [obj.to_json(run) for obj in threeD_list]
 
         for obj in jsons:
-            if not obj['path'].startswith(cls.get_media_subdir()):
+            expected = util.to_forward_slash_path(cls.get_media_subdir())
+            if not obj['path'].startswith(expected):
```

Comment:
This was a sick bug I caught because of our windows tests.  Object3D never worked in windows before 😭